README.md

```diff
@@ -1,19 +1,28 @@
 # cargo-semver-checks-action
 Lint your crate API changes for semver violations.
 
-```yaml
-- name: Check semver
-  uses: obi1kenobi/cargo-semver-checks-action@v2
-- name: Publish to crates.io
-  run: # your `cargo publish` code here
-```
-
+* [Basic usage](#basic-usage)
 * [Input options](#input-options)
 * [Example scenarios](#example-scenarios)
-  * [Use in workspaces with a single crate](#use-in-workspaces-with-a-single-crate)
-  * [Use in workspaces with more than one crate](#use-in-workspaces-with-more-than-one-crate)
+  * [Use outside root directory of a crate or a workspace](#use-outside-root-directory-of-a-crate-or-a-workspace)
+  * [Specify crates to be checked](#specify-crates-to-be-checked)
+  * [Exclude crates from being checked](#exclude-crates-from-being-checked)
+  * [Use toolchain other than `stable`](#use-toolchain-other-than-stable)
 * [Customizing baseline rustdoc caching strategy](#customizing-baseline-rustdoc-caching-strategy)
 
+## Basic usage
```

Comment:
Maybe mention here that the action is designed to run right before `cargo publish`, and explain what happens if you run it on PRs?

Reply:
I added some content in `Basic usage` in response to this and the below comment, please check if it is more descriptive now :).